pkgs/top-level/aliases.nix

```diff
@@ -878,6 +878,7 @@ mapAliases ({
     kwin-dynamic-workspaces
     kwin-tiling
     krohnkite
+    parachute
```

Comment:
We usually add aliases for backwards compatibility, so I don't we should add one for a new package.

Reply:
Fixed, thanks!